Sources/Tron/TronTransaction.swift

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import UIKit
+import Foundation
+
+enum TronTransactionError: Error {
+    case blockHeaderHeight
+    case blockHeaderRawData
```

Comment:
maybe `LocalizedError`? block height/raw data seems not like a error

Reply:
yeah it makes sense